### PR TITLE
World object fixes

### PR DESCRIPTION
**Pull request by Forbiddenz**
World Object Fixes + Start of Global Guid Manager

### CODE REVIEW

File: Source/ACE/Entity/Model.cs [hunk 1]
```diff
@@ -6,16 +6,26 @@
...
     public class Model
     {
-        public byte Index { get; } //index of model
 
-        public uint Guid { get; }  //- 0x01000000
+        /// <summary>
+        /// Index of Model being replace
+        /// </summary>
+        public byte Index { get; }
+        /// <summary>
```
Comment (Mogwai-AC) [inline Source/ACE/Entity/Model.cs]: empty line between fields please.

File: Source/ACE/Entity/ModelTexture.cs [hunk 2]
```diff
@@ -7,19 +7,28 @@
...
     /// </summary>
     public class ModelTexture
     {
-        public byte Index { get; } //index of model to replace texture.
-        public uint OldGuid { get; }
-        public uint NewGuid { get; }
+        /// <summary>
+        /// Index of model to replace texture.
+        /// </summary>
+        public byte Index { get; }
```
Comment (Mogwai-AC) [inline Source/ACE/Entity/ModelTexture.cs]: empty line between fields please

File: Source/ACE/Network/GameMessages/Messages/GameMessageCreateLifestone.cs [hunk 3]
```diff
@@ -31,7 +31,7 @@ public GameMessageCreateLifestone(Player player) : base(GameMessageOpcode.Object
             Position newPosition = new Position(player.Position.Cell, player.Position.Offset.X + dx, player.Position.Offset.Y + dy, player.Position.Offset.Z + 0.5f, 0f, 0f, 0f, 0f);
             
             var weenie = WeenieHeaderFlag.Usable | WeenieHeaderFlag.BlipColour | WeenieHeaderFlag.Radar | WeenieHeaderFlag.UseRadius;
-            ImmutableWorldObject wo = new ImmutableWorldObject(ObjectType.LifeStone, new ObjectGuid(nextObjectId++, GuidType.None), "Life Stone", ObjectDescriptionFlag.LifeStone, weenie, newPosition);
+            ImmutableWorldObject wo = new ImmutableWorldObject(ObjectType.LifeStone, new ObjectGuid(GuidManager.NewStaticObjectGuid()), "Life Stone", 1, ObjectDescriptionFlag.LifeStone, weenie, newPosition);
```
Comment (Mogwai-AC) [inline Source/ACE/Network/GameMessages/Messages/GameMessageCreateLifestone.cs]: If you're not going to implement WCID at this point, don't stub bad data.  We're close to using it properly.

File: Source/ACE/Network/Enum/PhysicsDescriptionFlag.cs [hunk 4]
```diff
@@ -23,7 +23,7 @@ public enum PhysicsDescriptionFlag
         DefaultScriptIntensity = 0x004000,
         Position               = 0x008000,
         Movement               = 0x010000,
-        AnimationFrame         = 0x200000,
+        AnimationFrame         = 0x020000,
         Translucency           = 0x400000
```
Comment (Mogwai-AC) [inline Source/ACE/Network/Enum/PhysicsDescriptionFlag.cs]: Og II also fixed this to have Translucency of 0x040000

File: Source/ACE/Entity/Player.cs [hunk 5]
```diff
@@ -197,7 +197,7 @@ public Player(Session session) : base(ObjectType.Creature, session.CharacterRequ
             GameData.Usable = Usable.UsableObjectSelf;
 
             SetPhysicsState(PhysicsState.IgnoreCollision | PhysicsState.Gravity | PhysicsState.Hidden | PhysicsState.EdgeSlide, false);
-            PhysicsData.PhysicsDescriptionFlag = PhysicsDescriptionFlag.CSetup | PhysicsDescriptionFlag.MTable | PhysicsDescriptionFlag.Stable | PhysicsDescriptionFlag.Petable | PhysicsDescriptionFlag.Position;
+            PhysicsData.PhysicsDescriptionFlag = PhysicsDescriptionFlag.CSetup | PhysicsDescriptionFlag.MTable | PhysicsDescriptionFlag.Stable | PhysicsDescriptionFlag.Petable | PhysicsDescriptionFlag.Position;// | PhysicsDescriptionFlag.Movement;
```
Comment (Mogwai-AC) [inline Source/ACE/Entity/Player.cs]: cleanup on aisle 200...

File: Source/ACE/Entity/MutableWorldObject.cs [hunk 6]
```diff
@@ -10,8 +10,8 @@ namespace ACE.Entity
     /// <summary>
     /// any world object that can change a state of some sort that requires clients be updated.  players, monsters,
     /// doors, etc.
-    /// </summary>
-    public abstract class MutableWorldObject : WorldObject
+     public abstract class MutableWorldObject : WorldObject/// </summary>
+
```
Comment (Mogwai-AC) [inline Source/ACE/Entity/MutableWorldObject.cs]: added whitespace, butchered the xml doc.  fix please.

File: Source/ACE/Network/GameMessages/Messages/GameMessageCreateLifestone.cs (hunk 3 above)
Comment (Forbiddenz) [inline Source/ACE/Network/GameMessages/Messages/GameMessageCreateLifestone.cs]: I am not sure how to fix it, but the way its in ace now its not named correctly.
Also, passing a 1, its required now.. now sure the proper way to use this.. but interested..
can we keep this now until its in properly ?

Comment (Mogwai-AC) [inline Source/ACE/Network/GameMessages/Messages/GameMessageCreateLifestone.cs]: i guess.  just another merge conflict for the landblock PR is all.

**Comment by Forbiddenz** [comment]
Cleaned up

**Comment by Mogwai-AC** [comment]
MutableWorldObject xml doc header....

**Pull request merged**